app/views/categories/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="form_container">
+  <div class="navbar">
+    <h2>New Category</h2>
+    <%= link_to categories_path do %>
+  <i class="fas fa-arrow-left"></i>
+  <span class="sr-only">Back</span>
+<% end %>
+
+  </div>
+
+  <%= form_for @category, html: { class: "navbar-form" } do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+    <%= f.label :icon %>
+    <%= f.text_field :icon %>
+    <%= f.submit "Save", class: "btn btn-primary" %>
+  <% end %>
```

Comment:
- [ ] Please fix the following error that occurs when the user tries to create a category. If you can't replicate this error. Please drop the database and start a newly created database, then run the project.

![image](https://github.com/danutibe07/BudgetApp/assets/62290545/5cd70518-3b4b-4383-9b92-2c2aa9420054)